packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -22,6 +25,7 @@ export interface ISelectProps
   onChange: React.ChangeEventHandler<HTMLSelectElement>;
   ref?: React.ForwardedRef<HTMLSelectElement>;
   ariaLabel: string;
+  variant?: (typeof variants)[number];
```

Comment:
You can remove "variants" and type it with just the variant object 
```suggestion
    variant?: keyof typeof selectVariants;
```